.github/workflows/pytest.yml

```diff
@@ -8,11 +8,16 @@ on:
       - opened
       - synchronize
       - ready_for_review
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
```

Comment:
Does `cancel-in-progress` mean that another push to the same PR will (say) kill an already running `ci-integration` job, and replace it with a new one?

Reply:
I think this is probably better. But we should make sure we tell everyone about the change since I've also definitely used the fact that they don't get killed to test a couple of things in parallel.